drivers/network/dd/e1000/hardware.c

```diff
@@ -12,7 +13,8 @@
 
 static USHORT SupportedDevices[] =
 {
-    0x100f,     // Intel 82545EM (VMWare E1000)
+    0x100e,     // Intel 82540EM
+    0x100f,     // Intel 82545EM
```

Comment:
Why remove the (VMWare E1000) part?

Reply:
82545EM is emulated not only in VMvare. I think, it is not consistent if we mention just VMware here. So I decided to remove the comment completely)